string package_name
string plugin_name
string[] parameters
---
bool success
